src/components/EmptyListMessage.spec.js

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import EmptyListMessage from './EmptyListMessage';
+
+describe('FormSubmitMessage', () => {
+  it('renders a message with text', () => {
+    render(
+      <MemoryRouter>
+        <EmptyListMessage />
+      </MemoryRouter>
+    );
+    // const test = screen.getByRole('');
+    const message = screen.getByText("You don't have any notes yet! What's on your mind? Write it down right here...");
+    expect(message).toBeInTheDocument();
+    const icon = screen.getByRole('img', { name: /feather/i });
+    expect(icon).toBeInTheDocument();
```

Comment:
The more complex your variables and `expect`expressions become, the harder this part can be read. I suggest to separate the variable declarations and expect part from each other.